docs/started/daemons.md

```diff
@@ -0,0 +1,91 @@
+---
+id: daemons
+title: Rucio Daemons
+sidebar_label: daemons
+---
+
+Rucio relies on several daemons (processes) to perform different logic.
+Most of the daemons connect to the DB to read some data, perform some computation, 
+and then write some data back into the DB.
+
+Usually one daemon will create some work for another daemon and viceversa.
+In Rucio realm, daemons communicate to others by the DB.
+
+The following table represents a high level view of the responsibility of each of the daemons.
+
+## Daemons
+
+Name | Domain|  Purpose |  Details
```

Comment:
I think this will be the last nit-pick. Is it is possible to set this up with the a relative path? Using this method will redirect the user to a different site if they are looking at the gh docs mirror.